templates/taxi/driver_list.html

```diff
@@ -8,6 +8,9 @@ <h1>Driver list</h1>
         <li>
           <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
```

Comment:
The URL tag uses double quotes for the URL name, which is acceptable but inconsistent with the single quotes used elsewhere in your templates. Consider using single quotes for consistency.